test/test_directive_define.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from otk.context import CommonContext
+from otk.error import TransformDirectiveTypeError
+from otk.directive import define
+
+
+def test_define():
+    ctx = CommonContext()
+
+    define(ctx, {"a": "b", "c": 1})
```

Comment:
This is really minor, but I find using 'b' for the value of 'a' makes it harder to read. If it were a word I wouldn't need to make sure that 'b' wasn't supposed to be a key.

Also, would it be worthwhile to have a key that doesn't get changed?